src/Accelerators/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if( ACCELERATORS_TO_BUILD )
+  foreach(t ${ACCELERATORS_TO_BUILD})
+    message(STATUS "Targeting ${t}")
```

Comment:
I would make this message either verbose or debug, so that it can be turned on easily, but not clutter the log otherwise.